lib/events/auditlog_test.go

```diff
@@ -464,6 +466,55 @@ func (a *AuditTestSuite) TestLegacyHandler(c *check.C) {
 	c.Assert(err, check.IsNil)
 }
 
+func (a *AuditTestSuite) TestConcurrentStreaming(c *check.C) {
```

Comment:
I think RJ already converted this to standard go tests on v10 and master, but didn't backport to v9.

I started on v9 because that's the version the customer is on who experienced this issue. It was easiest to just follow the existing pattern here, and I'm planning to use standard Go testing when I forward-port to v10 and master.

That said, this ended up being pretty self contained, so I'm happy to put it in it's own non-check test right now if you'd prefer.